### DISCUSSION TITLE
FluentNavLink format

### DISCUSSION THREAD

**Discussion by simonunivonix** [microsoft/fluentui-blazor#2363] [Q&A]
1) Is it possible to change (specifically decrease) the indentation of FluentNavLinks that are part of a FluentNavGroup?
2) Is it possible to change the style of the text of the selected FluentNavLink so that it will be bold and the same color as the verticle bar that indicates that the FluentNavLink  is selected (this can be different depending on the theme chosen).  Obviously, when unselecting the text style should return to the default state?

**Comment by vnbaaij** [comment]
Yes, that is all possible. It is just html and css in the end. But we don't really expose every one of those options explicitly because we have designed a specific look and feel.
You'll need to read up on shadow dom and part etc. And also search through the closed issues. A lot of examples have been discussed before.
